spec/ddtrace/tracer_integration_spec.rb

```diff
@@ -76,4 +76,53 @@ def lang_tag(span)
       it { expect(@child_root_span).to be child_span }
     end
   end
+
+  context 'with synthetics' do
```

Comment:
I think it's a good idea to have a few top level integration tests like this: this bug manifested because we never actually had any basic tests to verify the most simple happy path for synthetics.

In the future, I'd like to extract integration tests like this to their own suite of sorts, separate from unit tests, as each kind of test serves an important purpose, but integration tests like this have a habit of creeping into the space unit tests should possess.